src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -1,12 +1,14 @@
 import { useFocusEffect, useRoute } from '@react-navigation/core';
 import lang from 'i18n-js';
+import { isEmpty } from 'lodash';
```

Comment:
i think we want to stop using lodash

Reply:
Interesting, is there a convo somewhere about this? Just curious, still learning about the codebase 😛

Reply:
I don't think we have an `isEmpty` replacement right now. Can we make that a separate PR?

Reply:
@estrattonbailey we're just trying to move away from lodash but jake is right, isEmpty hasn't been replaced yet